requestor/yagna_erigon_manager.py

```diff
@@ -5,23 +5,51 @@
 from yapapi import Executor, Task
 from datetime import timedelta
 from worker import worker
+from dataclasses import dataclass, field
+from typing import Optional
+from datetime import datetime
+
 
 SUBNET_TAG = 'ttt'
 
 
+@dataclass
+class RuntimeState():
+    status: str
+    url: Optional[str] = None
+    secret: Optional[str] = None
+    timestamp: datetime = field(init=False)
+
+    def __post_init__(self):
+        self.timestamp = datetime.now()
+
+
 class Erigon():
     def __init__(self):
         self.id = self._create_id()
         self.queue = asyncio.Queue()
         self.start_fut = None
         self.stopped = False
+        self.runtime = RuntimeState('initializing')
+        self.update_task = asyncio.create_task(self.update_state())
+
+    async def update_state(self):
+        while True:
+            if self.stopped:
+                self.runtime = RuntimeState('stopping')
+                break
+            res = await self.status()
+            self.runtime = RuntimeState(**res)
+            await asyncio.sleep(1)
```

Comment:
1 second interval seems a little bit too often for me. I'd suggest 3–5 secs. Also, this may be moved to a constant, or even better, to an environment variable with a default value.

Reply:
I added the constant - it is set to 1 now (for the development purposes).

About environment variable: there are other configuration options (subnet? payment driver?) that also have to be dealt with so I think this should wait.